contributions/feedback/heeenkie/README.md

```diff
@@ -0,0 +1,8 @@
+# Feedback on BDD in DevOps #
+
+# Members #
+Andreas Henriksson (anhenri@kth.se), Github: [heeenkie](https://github.com/heeenkie)
+
+#1032
```

Comment:
Please link to the PR like so:
[#1032](https://github.com/KTH/devops-course/pull/1032)

Reply:
@Deee92 Done.